app/population/index.js

```diff
@@ -1,20 +1,30 @@
 const router = require('express').Router()
+const dailyRouter = require('express').Router({ mergeParams: true })
 
 const {
   setContext,
   setDateRange,
   setPagination,
 } = require('../../common/middleware/collection')
 
-const { BASE_PATH, MOUNTPATH } = require('./constants')
-const { dashboard } = require('./controllers')
-const { setResultsAsPopulationTable, redirectBaseUrl } = require('./middleware')
+const { BASE_PATH, MOUNTPATH, DAILY_PATH } = require('./constants')
+const { dashboard, daily } = require('./controllers')
+const {
+  redirectBaseUrl,
+  setPopulationId,
+  setResultsAsPopulationTable,
+} = require('./middleware')
 
 router.param('date', setDateRange)
+router.param('locationId', setPopulationId)
 
 router.get('/', redirectBaseUrl)
 
-router.use(
+dailyRouter.get('/', setContext('population'), daily)
```

Comment:
I think you can remove `setContext` now as it's no longer being used in this view.

Reply:
Removed.